hazelcast/src/test/java/com/hazelcast/spi/impl/operationservice/impl/InvocationFutureTest.java

```diff
@@ -36,91 +36,52 @@ public void setup() {
         operationService = getOperationService(local);
     }
 
-    // ================= isDone ==========================================
-
     @Test
     public void isDone_whenNullResponse() throws ExecutionException, InterruptedException {
         DummyOperation op = new DummyOperation(null);
 
         InternalCompletableFuture future = operationService.invokeOnTarget(null, op, getAddress(local));
-        // first we wait for the future to complete.
-        future.get();
+        future.get(); // first we wait for the future to complete.
 
         assertTrue(future.isDone());
     }
 
     @Test
     public void isDone_whenWaitResponse() {
-        DummyOperation op = new DummyOperation() {
-            {
-                // we need to set the call-id to prevent running the operation on the calling-thread.
-                setPartitionId(1);
-            }
-
-            @Override
-            public void run() throws Exception {
-                Thread.sleep(5000);
-            }
-        };
+        DummyOperation op = new DummySleepingNullReturningOperation();
```

Comment:
The problem with DummyPartitionOperation/TestPartitionOperation names is that they all give the impression that they matter. In our case, the GetLostOperation is meant to create an invocation and to shoot it so much into the future that it won't bother the actual test. So I don't care about its results.. I just want to get access to an Invocation instance I can test.